changelogs/unreleased/pagination-fix-iso6.yml

```diff
@@ -0,0 +1,3 @@
+description: "Fix for pagination in Service Inventory + tests"
```

Comment:
I think it's not the idea to hide all our changelogs in the output. Only the ones that treat a same subject like the Composer, should be grouped under one visible changelog to avoid having twenty five logs about the same thing. Bugfixes, and other more generic issues can have a full changelog.